Tasks/NuGetCommand/nugetcommandmain.ts

```diff
@@ -52,4 +55,81 @@ async function main(): Promise<void> {
+                'configurationToPack': tl.getInput('configurationToPack'),
+                'versioningScheme': tl.getInput('versioningScheme'),
+                'includeReferencedProjects': tl.getBoolInput('includeReferencedProjects'),
+                'versionEnvVar': tl.getInput('versionEnvVar') ?
+                    tl.getInput(tl.getInput('versionEnvVar')) : null,
+                'requestedMajorVersion': tl.getInput('requestedMajorVersion'),
+                'requestedMinorVersion': tl.getInput('requestedMinorVersion'),
+                'requestedPatchVersion': tl.getInput('requestedPatchVersion'),
+                'packTimezone': tl.getInput('packTimezone'),
+                'buildProperties': tl.getInput('buildProperties'),
+                'verbosityPack': tl.getInput('verbosityPack'),
+                'includeSymbols': tl.getBoolInput('includeSymbols'),
+                'NuGet.UseLegacyFindFiles': tl.getVariable('NuGet.UseLegacyFindFiles'),
+                'NuGetTasks.IsHostedTestEnvironment': tl.getVariable('NuGetTasks.IsHostedTestEnvironment'),
+                'System.TeamFoundationCollectionUri': tl.getVariable('System.TeamFoundationCollectionUri'),
+                'NuGet.OverwritePackagingCollectionUrl': tl.getVariable('NuGet.OverwritePackagingCollectionUrl'),
+                'externalendpoint': externalendpoint,
+                'externalendpoints': externalendpoints,
+                'allowpackageconflicts': tl.getInput('allowpackageconflicts'),
+                'configurationtopack': tl.getInput('configurationtopack'),
+                'includenugetorg': tl.getInput('includenugetorg'),
+                'includereferencedprojects': tl.getInput('includereferencedprojects'),
+                'includesymbols': tl.getInput('includesymbols'),
+                'nocache': tl.getInput('nocache'),
+                'nugetconfigpath': tl.getInput('nugetconfigpath'),
+                'nugetfeedtype': tl.getInput('nugetfeedtype'),
+                'outputdir': tl.getInput('outputdir'),
+                'packtimezone': tl.getInput('packtimezone'),
+                'requestedmajorversion': tl.getInput('requestedmajorversion'),
+                'requestedminorversion': tl.getInput('requestedminorversion'),
+                'requestedpatchversion': tl.getInput('requestedpatchversion'),
+                'searchpatternpack': tl.getInput('searchpatternpack'),
+                'searchpatternpush': tl.getInput('searchpatternpush'),
+                'selectorconfig': tl.getInput('selectorconfig'),
+                'solution': tl.getInput('solution'),
+                'versioningscheme': tl.getInput('versioningscheme'),
+                'verbositypack': tl.getInput('verbositypack'),
+                'verbositypush': tl.getInput('verbositypush'),
+                'verbosityrestore': tl.getInput('verbosityrestore')
+            });
+
+        let nugetTelem = JSON.parse(nugetTelemStr);
```

Comment:
Same as other comment, parse is not needed if not stringify first.

Reply:
Fixed